src/core/Link/Link/Link.md

```diff
@@ -45,12 +45,22 @@ import { SkipLink } from 'suomifi-ui-components';
 ```js
 import { ExternalLink } from 'suomifi-ui-components';
 
-<ExternalLink
-  href="https://www.sweden.se/"
-  labelNewWindow="Opens to a new window"
->
-  External link
-</ExternalLink>;
+<>
+  <ExternalLink
+    href="https://designsystem.suomi.fi/fi/"
+    toNewWindow
+    labelNewWindow="Opens to a new window"
+  >
+    External link
+  </ExternalLink>
+  <ExternalLink
+    href="https://designsystem.suomi.fi/fi/"
+    toNewWindow={false}
+    labelNewWindow="Opens to same window"
+  >
+    External link
+  </ExternalLink>
```

Comment:
Should be wise to somehow separate the examples with different texts.

Reply:
I agree. Other than this the change looks good :)

Reply:
Fixed in md-file